setup/.setuptools-odoo-make-default-ignore

```diff
@@ -0,0 +1,2 @@
+# addons listed in this file are ignored by
```

Comment:
just FYI setup files should be generated by bot, there is no need to add them

Reply:
this file generated by pre-commit. if i don't add it, it will not travis green.